src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -325,6 +342,8 @@ const BaseComment = (props: BaseCommentProps) => {
                     {...commentProps}
                     isParentPending={isPending}
                     isRepliesLoading={isRepliesLoading}
+                    // note that passing down onCommentEdit does not allow editing replies
+                    onEdit={noop}
```

Comment:
sorry unfamiliar here: are replies not editable?

why do we pass a prop to `Replies` if it's just a `noop` instead of passing the `noop` to `BaseComment` directly in the `Replies` implementation?